azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeLocalOptions.java

```diff
@@ -234,4 +237,24 @@ public CallCompositeLocalOptions setCaptionsOptions(
         this.captionsOptions = captionsOptions;
         return this;
     }
+
+    /* <END_CALL_FOR_ALL>
+    \**
+     * Get the boolean value for on call end terminate for all.
```

Comment:
I do not think 'boolean value' is necessary. 
How about 'Configures if call should end for all participants when current user hangs up. '